dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -85,7 +99,23 @@ def self.infer_no(state_code)
     INFERRED_NO_EXCLUSION_LIST.exclude? state_code.upcase
   end
 
-  def self.construct_state_school_id(state_code, row_hash)
+  def self.construct_state_school_id(state_code, row_hash, school_year)
```

Comment:
Would it be worth renaming this method to `get_state_school_id` since that’s what we are typically doing here?  In some rare/legacy cases, we are constructing the state_school_id from district_id and school_id.

Reply:
I like this also because it avoids duplicating the function name in `School` that we call a lot in here.